vmdb/app/models/ems_refresh/parsers/ec2.rb

```diff
@@ -465,6 +465,9 @@ def find_stack_resources(stack)
       # convert the AWS Resource Summary collection to an array to avoid the same API getting called twice
       raw_resources = stack.resource_summaries.to_a
 
+      # physical_resource_id can be empty if the resource was not successfully created; ignore such
+      raw_resources.reject! { |r| r[:physical_resource_id].nil? }
```

Comment:
This problem was discovered when an exception was raised at saving inventory. If we don't ignore it here, `raw_resources.collect` block a few lines below will create an empty element, which is not desired.